serverless/pytorch/dschoerk/transt/nuclio/main.py

```diff
@@ -0,0 +1,145 @@
+import json
+import base64
+from PIL import Image
+import io
+import numpy as np
+import traceback
+import jsonpickle
+import torch
+
+from pysot_toolkit.bbox import get_axis_aligned_bbox
+from pysot_toolkit.trackers.tracker import Tracker
+from pysot_toolkit.trackers.net_wrappers import NetWithBackbone
+
+def create_tracker():
+    use_gpu = torch.cuda.is_available()
+
+    net_path = '/transt.pth'  # Absolute path of the model
+    net = NetWithBackbone(net_path=net_path, use_gpu=use_gpu)
+    tracker = Tracker(name='transt', net=net, window_penalty=0.49, exemplar_size=128, instance_size=256)
+    return tracker
+
+def init_tracker(tracker, img, bbox):
+    cx, cy, w, h = get_axis_aligned_bbox(np.array(bbox))
+    gt_bbox_ = [cx - w / 2, cy - h / 2, w, h]
+    init_info = {'init_bbox': gt_bbox_}
+    tracker.initialize(img, init_info)
+
+    return tracker
+
+def track(tracker, img):
+    outputs = tracker.track(img)
+    prediction_bbox = outputs['target_bbox']
+
+    left = prediction_bbox[0]
+    top = prediction_bbox[1]
+    right = prediction_bbox[0] + prediction_bbox[2]
+    bottom = prediction_bbox[1] + prediction_bbox[3]
+    return tracker, (top, left, bottom, right)
+
+
+
+def init_context(context):
+    context.logger.info("Init context...  0%")
+    model = create_tracker()
+    context.user_data.model = model
+    context.logger.info("Init context...100%")
+
+def log(msg):
+    #with open("/log.log", "a") as logf:
+    #    logf.write(msg+'\n')
+    pass
+
+def encode_state(model):
```

Comment:
Could you please separate these functions into a separate `ModelHandler` class (as is done for other serverless functions)?